backend/pong/game_status.py

```diff
@@ -0,0 +1,80 @@
+from .constants import *
+import math
+from .game_logic import PongGame
+
+
+class PongStatus:
+    def __init__(self):
+        # Initialize game state
+        self.player_left_y = PLAYGROUND_HEIGHT // 2 
+        self.player_right_x = PLAYGROUND_WIDTH - WALL_MARGIN
+        self.player_left_x = WALL_MARGIN
+        self.player_right_y = PLAYGROUND_HEIGHT // 2
```

Comment:
I'd still want the plaýer, the ball and the overall game stats to be separated in their own structs.